test/e2e/transpile-packages/index.test.ts

```diff
@@ -18,16 +17,6 @@ describe('transpile packages', () => {
       dependencies: {
         sass: 'latest',
       },
-      packageJson: {
```

Comment:
This is the default so I'm not sure why it was added, maybe a legacy thing when we used yarn.